src/components/MoreInformation.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+	<div class="more-information-overlay">
+		<div class="more-information-card" @click.stop>
+			<div class="x-image-container">
+				<img class="x-image-container" alt="exit" src="../assets/x-close.png" @click="closeModal"/>
+			</div>
+			<img :src="sprite" alt="pokemon"/>
+			<p class="poke-name">Name: {{ name }}</p>
+			<div class="list">
+				Abilities:
+				<div>
+					<span v-for="(ability, index) in abilities">Ability {{ index + 1 }} : {{ ability.ability.name }}</span>
+				</div>
+			</div>
+			<div class="list">
+				Stats:
+				<div>
+					<p v-for="(stat) in stats" class="stats-list">
+						<span>{{ stat.stat }}</span>
+					</p>
+				</div>
+			</div>
+			<Button class="button-align" @click="closeModal">CLOSE</Button>
+		</div>
+	</div>
+</template>
+
+<script lang="ts" setup>
+import { Pokemon } from '../interfaces'
+import Button from './Button.vue'
+import { capitalize } from '../Common/capitalize'
+import { idGenerator } from '../Common/idGenerator'
+
+
+const props = defineProps<{
+	pokemon: Pokemon
+}>()
+
+const generateId = idGenerator()
+const { ...pokemon }: Pokemon = props.pokemon
+const emits = defineEmits(['close'])
+
+function closeModal(){
+	emits('close')
+}
+
+const stats = pokemon.stats.reduce((acum, item) => [...acum, { stat: `${item.stat.name}: ${item.base_stat}` }], [])
+const abilities = pokemon.abilities
+const type = pokemon.types[0].type.name
+const name = capitalize(pokemon.name)
+const sprite = pokemon.sprites['official-artwork'].front_default
```

Comment:
Looks like this code is repeated from [pokemonPropertiersStractor.ts](https://github.com/RenanPaixao/Apokeose/pull/1/files#diff-865947c5c9b5e7cd0f612e608ecc0c4129ab6090f306489ac3a17486ea7bf32b)